tests/testthat/testImportNetMultinet.r

```diff
@@ -0,0 +1,200 @@
+                               verbose = TRUE,
+                               silent = TRUE,
+                               writeLog = FALSE),
+      '.checkArgs> "fName" mode error')
+    
+    expect_error(
+      gG <- importNet.Multinet(fName = c("fee", "fie", "fo", "fum"),
+                               net = "PPI",
+                               verbose = TRUE,
+                               silent = TRUE,
+                               writeLog = FALSE),
+      '.checkArgs> "fName" length error')
+    
+    expect_error(
+      gG <- importNet.Multinet(fName = "nonSuch.txt",
+                               net = "PPI",
+                               verbose = TRUE,
+                               silent = TRUE,
+                               writeLog = FALSE),
+      '.checkArgs> "fName" error: file "nonSuch.txt" does not exist.')
+    
+    # ==== net
+    expect_error(
+      gG <- importNet.Multinet(fName = fN,
+                               net = NULL,
+                               verbose = TRUE,
+                               silent = TRUE,
+                               writeLog = FALSE),
+      '.checkArgs> "net" mode error')
+    
+    expect_error(
+      gG <- importNet.Multinet(fName = fN,
+                               net = c("PPI", "GENETIC"),
+                               verbose = TRUE,
+                               silent = TRUE,
+                               writeLog = FALSE),
+      '.checkArgs> "net" length error')
+    
+    expect_error(
+      gG <- importNet.Multinet(fName = fN,
+                               net = "noSuchNet",
+                               verbose = TRUE,
+                               silent = TRUE,
+                               writeLog = FALSE),
+      "Request error:")
+    
+    # ==== verbose
+    expect_error(
+      gG <- importNet.Multinet(fName = fN,
+                               net = "PPI",
+                               verbose = NULL,
+                               silent = TRUE,
+                               writeLog = FALSE),
+      '.checkArgs> "verbose" mode error')
+    
+    # ==== silent
+    expect_error(
+      gG <- importNet.Multinet(fName = fN,
+                               net = "PPI",
+                               verbose = TRUE,
+                               silent = NULL,
+                               writeLog = FALSE),
+      '.checkArgs> "silent" mode error')
+    
+    # ==== writeLog
+    expect_error(
+      gG <- importNet.Multinet(fName = fN,
+                               net = "PPI",
+                               verbose = TRUE,
+                               silent = TRUE,
+                               writeLog = NULL),
+      '.checkArgs> "writeLog" mode error')
+    
+    # ==== no logfile should have been written
+    expect_false(file.exists(logName))
+    
+})
+
+
+test_that("importNet.Multinet() produces gG from Multinet data", {
+    fN <- "Multinet.interactions.network_presence.txt"
```

Comment:
Where is this file located? You need a small(!) excerpt as test-data, not the whole thing! The test-data can be stored in this directory, or in ```inst/extdata/```. Check other functions' use of test data files for examples.